libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -628,6 +632,15 @@ singleton AP_Winch method release_length void float'skip_check
 singleton AP_Winch method set_desired_rate void float'skip_check
 singleton AP_Winch method get_rate_max float
 
+include AP_IOMCU/AP_IOMCU.h
+singleton AP_IOMCU depends HAL_WITH_IO_MCU
+singleton AP_IOMCU rename iomcu
+singleton AP_IOMCU method healthy boolean
+
+include AP_Compass/AP_Compass.h
+singleton Compass rename compass
+singleton Compass method healthy boolean uint8_t 0 COMPASS_MAX_INSTANCES
```

Comment:
It would be nice to move away from limiting to `COMPASS_MAX_INSTANCES` but I think if you went outside that range bad things would happen.

The problem is that the user has no way to tell what `COMPASS_MAX_INSTANCES` is from the script, if they try and read a instance that is too high it will crash the script. Instead we would prefer to allow any index to be asked for and then return null if its invalid. 

There is the same issue on the `AP_InertialSensor` bindings of course, but your keeping in line with the existing pattern there.

Reply:
Yeah, i didn't quite understand what would be best to put there, i copied how simular bindings were added. The other option that is available here is to add another binding for https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Compass/AP_Compass.h#L156
```suggestion
singleton Compass method healthy boolean uint8_t 0 COMPASS_MAX_INSTANCES
singleton Compass method get_count uint8_t
```

Reply:
This should be resolved now https://github.com/ArduPilot/ardupilot/pull/23634/commits/9612dd8a4dee107ce8e630831b83a29e347c43ac :)